diff --git a/src/Plugin/CKEditorPlugin/YoutubePlugin.php b/src/Plugin/CKEditorPlugin/YoutubePlugin.php
index 81d24e8..b54b8d5 100755
--- a/src/Plugin/CKEditorPlugin/YoutubePlugin.php
+++ b/src/Plugin/CKEditorPlugin/YoutubePlugin.php
@@ -22,7 +22,7 @@ class YoutubePlugin extends CKEditorPluginBase {
     return array(
       'Youtube' => array(
         'label' => t('YouTube'),
-        'image' => 'libraries/youtube/images/icon.png',
+        'image' => 'libraries/youtube/youtube/images/icon.png',
       ),
     );
   }
@@ -52,7 +52,7 @@ class YoutubePlugin extends CKEditorPluginBase {
    * {@inheritdoc}
    */
   public function getFile() {
-    return 'libraries/youtube/plugin.js';
+    return 'libraries/youtube/youtube/plugin.js';
   }
 
   /**
